### DISCUSSION TITLE
depth of field oddity, is this a bug?

### DISCUSSION THREAD

**Discussion by jwwalker** [google/filament#3781] [Q&A] [answered]
In the `gltf_viewer` sample, if you turn on DoF and then move the focus distance way down, say 0.23 and below, then the object looks not only blurry but larger.  Would that be considered a bug, or is it just a case where you can't be perfectly accurate in all edge cases?

**Comment by romainguy** [comment, marked as answer]
@pixelflinger Is that the "focus distance affects FoV" effect you implemented? (as in real cameras). It might also just be an "illusion" because of the intense blur radius.

**Reply by pixelflinger** [reply]
Ah yes, it's because I implemented the "effective focal length" in gltf-viewer. It's not something filament does, it's just something gltf-viewer does. It emulates (certain) real cameras. Maybe that should be an option, or we should just turn it off.
